javascript/javascript1/week1/eznamey.js

```diff
@@ -0,0 +1,5 @@
+const firstWords = ["fabulous", "amazing", "awesome", "great", "wonderful", "genius", "creative", "incredible", "stunning"];
+const secondWords = ["lab", "studio", "hub", "works", "forge", "factory", "nest", "hive", "vault", "collective"];
+const randomNumber = Math.floor(Math.random() * 10);
```

Comment:
Use two random numbers to access the arrays. With this solution the words will always come out in parallel, so if the first word is Awesome, the second one will always be hub.